modular_skyrat/modules/customization/modules/clothing/under/sashes.dm

```diff
@@ -0,0 +1,192 @@
+	name = "chemist's sash"
+	icon_state = "chemist"
+
+/obj/item/clothing/under/bodysash/virologist
+	name = "virologist's sash"
+	icon_state = "virologist"
+
+/obj/item/clothing/under/bodysash/paramedic
+	name = "paramedic's sash"
+	icon_state = "paramedic"
+
+//Engineering
+/obj/item/clothing/under/bodysash/ce
+	name = "chief engineer's sash"
+	icon_state = "ce"
+
+/obj/item/clothing/under/bodysash/engineer
+	name = "engineer's sash"
+	icon_state = "engineer"
+
+/obj/item/clothing/under/bodysash/atmos
+	name = "atmospherics technician's sash"
+	icon_state = "atmos"
+
+//Science
+/obj/item/clothing/under/bodysash/rd
+	name = "research director's sash"
+	icon_state = "rd"
+
+/obj/item/clothing/under/bodysash/scientist
+	name = "scientist's sash"
+	icon_state = "scientist"
+
+/obj/item/clothing/under/bodysash/roboticist
+	name = "roboticist's sash"
+	icon_state = "roboticist"
+
+/obj/item/clothing/under/bodysash/geneticist
+	name = "geneticist's sash"
+	icon_state = "geneticist"
+
+//Supply/Civilian
+/obj/item/clothing/under/bodysash/hop
+	name = "head of personnel's sash"
+	icon_state = "hop"
+
+/obj/item/clothing/under/bodysash/qm
+	name = "quarter master's sash"
+	icon_state = "qm"
+
+/obj/item/clothing/under/bodysash/cargo
+	name = "cargo technician's sash"
+	icon_state = "cargo"
+
+/obj/item/clothing/under/bodysash/miner
+	name = "shaft miner's sash"
+	icon_state = "miner"
+
+/obj/item/clothing/under/bodysash/clown
+	name = "clown's sash"
+	icon_state = "clown"
+
+/obj/item/clothing/under/bodysash/mime
+	name = "mime's sash"
+	icon_state = "mime"
+
+/obj/item/clothing/under/bodysash/prisoner
+	name = "prisoner's sash"
+	icon_state = "prisoner"
+
+/obj/item/clothing/under/bodysash/cook
+	name = "cook's sash"
+	icon_state = "cook"
+
+/obj/item/clothing/under/bodysash/bartender
+	name = "bartender's sash"
+	icon_state = "bartender"
+
+/obj/item/clothing/under/bodysash/chaplain
+	name = "chaplain's sash"
+	icon_state = "chaplain"
+
+/obj/item/clothing/under/bodysash/curator
+	name = "curator's sash"
+	icon_state = "curator"
+
+/obj/item/clothing/under/bodysash/lawyer
+	name = "lawyer's sash"
+	icon_state = "lawyer"
+
+/obj/item/clothing/under/bodysash/botanist
+	name = "botanist's sash"
+	icon_state = "botanist"
+
+/obj/item/clothing/under/bodysash/janitor
+	name = "janitor's sash"
+	icon_state = "janitor"
+
+/obj/item/clothing/under/bodysash/psychologist
+	name = "psychologist's sash"
+	icon_state = "psychologist"
+
+//CentCom
+/obj/item/clothing/under/bodysash/centcom
+	name = "centcom's sash"
+	icon_state = "centcom"
+
+/obj/item/clothing/under/bodysash/official
+	name = "official's sash"
+	icon_state = "official"
+
+/obj/item/clothing/under/bodysash/intern
+	name = "intern's sash"
+	icon_state = "intern"
+
+/obj/item/clothing/under/bodysash/civilian
+	name = "civilian's sash"
+	icon_state = "civilian"
+
+///Misc
+
+/obj/item/clothing/under/bodysash/russia
+	name = "Russian sash"
+	icon_state = "russia"
```

Comment:
This sash isn't used anywhere unless you plan on porting the ruin for it.

Reply:
may as well still have the item